tchannel/tornado/dispatch.py

```diff
@@ -70,50 +71,53 @@ def __init__(self, _handler_returns_response=False):
         self.register(self.FALLBACK, self.not_found)
         self._handler_returns_response = _handler_returns_response
 
-    _HANDLER_NAMES = {
-        Types.CALL_REQ: 'pre_call',
-        Types.CALL_REQ_CONTINUE: 'pre_call'
-    }
+        self._HANDLERS = {
+            Types.CALL_REQ: self.handle_call_req,
+            Types.CALL_REQ_CONTINUE: self.handle_call_req_cont,
+        }
```

Comment:
No. How you'll do this is:

    def foo(self, arg1, arg2):
        # ...

    HANDLERS = {'x': foo}

Then to call it,

    HANDLERS['x'](self, arg1, arg2)